libs/libnx/nxmu/nx_eventhandler.c

```diff
@@ -143,6 +143,10 @@ int nx_eventhandler(NXHANDLE handle)
                 }
             }
         }
+      else if (nbytes == 0)
+        {
+          return OK;
```

Comment:
Why we return `OK`, but not `ERROR` with `_NX_SETERRNO(EBADMSG);`?

Reply:
emm... This PR just to resolve the compile issue, I do not want to change the original logic of this file, I am worried that if return error, it may cause other issues....

Reply:
or `ENOMSG`

Reply:
I think that the nature of an error is that `DEBUGASSERT(nbytes >= sizeof(struct nxclimsg_s));` is not hit if `CONFIG_DEBUG_ASSERTIONS` is off, but maybe I'm wrong. Maybe it is reasonable to add
```
if (nbytes < sizeof(struct nxclimsg_s))
  {
    _NX_SETERRNO(EBADMSG);
   return ERROR;
  }
```
instead of this change? What do you think?